meal.go

```diff
@@ -93,21 +94,25 @@ func GetMealIds(c *gin.Context) {
 	})
 }
 
+// 실제 식단 데이터
 type Diet struct {
 	Diet    string `json:"diet"`
 	Calorie string `json:"calorie"`
 }
 
+// 점심 학식
 type Lunch struct {
 	A Diet `json:"a"`
 	B Diet `json:"b"`
 	C Diet `json:"c"`
 }
 
+// 저녁 학식
 type Dinner struct {
 	A Diet `json:"a"`
 }
 
+// 학식 데이터
```

Comment:
Codacy found an issue: [comment on exported type MealData should be of the form "MealData ..." (with optional leading article)](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091160)